docs/index.md

```diff
@@ -0,0 +1,45 @@
+The TLS Certificates Operator charm is responsible for distributing certificates through relationships. Certificates are provided by the operator through Juju configs.
+
+## Usage
+To deploy the TLS Certificates Operator, all you need to do is run the following command, which will fetch the charm from [Charmhub](https://charmhub.io/tls-certificates-operator) and deploy it to your model:
+```shell
+juju deploy tls-certificates-operator
+```
+
+Juju will now fetch TLS Certificates Operator and begin deploying it to the local MicroK8s. This process can take several minutes depending on how provisioned (RAM, CPU, etc) your machine is. You can track the progress by running:
+```shell
+juju status --watch 1s
+```
+The deployment of the charm is completed when its status is in `Active/Idle` state.
+
+At this point you can relate the TLS Certificates Operator to your charm through the `tls-certificates` interface:
```

Comment:
Relate to a requirer of TLS certificates:

Reply:
Done.